src/app/services/authentication.service.ts

```diff
@@ -12,25 +12,15 @@ const httpOptions = {
 })
 export class AuthenticationService {
   userStatus = false;
-  key = 'token';
   constructor(private http: HttpClient, private appService: AppService) { }
    login(username: string, password: string) {
      if (username === 'mifos' && password === 'password') {
```

Comment:
@CajetanRodrigues hardcoded, intergrate APIs.

Reply:
The Backend team has to add CORS configuration on the entity in spring, and therefore I cannot access it. 
I have opened an issue on the mailing list.
For the time being, I have added those mock credentials, we can change that in the future.

Reply:
Hi Ankit, I have incorporated all the changes you have suggested.